poi/src/main/java/org/apache/poi/hpsf/TypedPropertyValue.java

```diff
@@ -255,7 +255,7 @@ static void skipPadding( LittleEndianByteArrayInputStream lei ) {
         for (int i=0; i<skipBytes; i++) {
             lei.mark(1);
             int b = lei.read();
-            if (b == -1 || b != 0) {
+            if (b != 0) {
```

Comment:
I think it obvious here... whenever `b == -1` there always be `b != 0`

Reply:
committed this one just now